tools/sgcollect_info

```diff
@@ -35,7 +35,7 @@ from tasks import AllOsTask
 from tasks import make_curl_task
 from tasks import flatten
 from tasks import do_upload_and_exit
-from tasks import add_file_task
+from tasks import add_file_task, add_gzip_file_task
```

Comment:
For consistency, should the new import be on its own line?  Or was this intentional because of the functional similarity?

Reply:
Sure, no reason why it shouldn't have it's own line